pybliometrics/scopus/utils/parse_content.py

```diff
@@ -2,6 +2,10 @@
 from functools import reduce
 from warnings import warn
 
+def html_unescape(s:str): 
+        from html import unescape
+        return unescape(s) if s else None
+
```

Comment:
Please sort the function alphabetically and make sure to have two newlines before and after